### PR TITLE
Added test for change password

### PR DESCRIPTION
**Pull request by Valekus**

### CODE REVIEW

File: .idea/libraries/selenium_java_3_12_0.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<component name="libraryTable">
```
Comment (archick12) [inline .idea/libraries/selenium_java_3_12_0.xml:1]: This file shouldn't be present in the master

File: .idea/libraries/testng_6_7.xml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<component name="libraryTable">
```
Comment (archick12) [inline .idea/libraries/testng_6_7.xml:1]: .idea folder must be excluded from the commit.

File: src/main/java/Pages/ChangePasswordPopUp.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+
+public class ChangePasswordPopUp {
+    public ChangePasswordPopUp(WebDriver driver){this.driver = driver;}
+        WebDriver driver;
+    String fulHeaderButtonId = "//header/nav/div/div[3]/ul/li[4]/a/span/span/img";
+    String profileId = "//*[@id=\"view_profile\"]";
+    String changePassId = "//*[@id=\"view_change_password\"]";
```
Comment (archick12) [inline src/main/java/Pages/ChangePasswordPopUp.java]: Remove escaping character \" with '

**Review by archick12 — CHANGES_REQUESTED** [review]

File: .idea/libraries/selenium_java_3_12_0.xml (hunk 1 above)
Comment (Valekus) [reply .idea/libraries/selenium_java_3_12_0.xml:1]: Added file to .gitignore

File: .idea/libraries/testng_6_7.xml (hunk 2 above)
Comment (Valekus) [reply .idea/libraries/testng_6_7.xml:1]: added file to .gitignore

**Pull request merged**